bin/yaml/libraries.yaml

```diff
@@ -237,6 +237,7 @@ libraries:
         - hwy_contrib
       targets:
         - 0.12.2
+        - trunk
```

Comment:
Rather unfortunately trunk has to be done separately (to thwart/workaround the check we do to not re-do existing work). I'll update manually and CC you on the change for future reference. This is on us: sorry!